_posts/2016-04-22-middle-aged-CMS-glitch.md

```diff
@@ -0,0 +1,97 @@
+---
+
+title: 'MIDDLE AGED CMS GLITCH: TGIF(80)'
+author: Cecily
+layout: blog-post-tgif
+comments: false
+
+---
+
+
+TGIF! – Thank Grohl, Gauss, Glob, Galileo and Science it’s Friday, and again it's time for our weekly reading recommendations, prepared to help you get some good reading time on your weekend. As usually, they're curated for you by [Cecily](https://twitter.com/skeskali)
+
+Oh, and by the way: <b>our blog's RSS feed is back</b> – subscribe now on hood.ie or hood.ie/feed.xml to make sure you never miss a post! <3
+
+
+
+## Open Source
+
+<blockquote class="twitter-tweet" data-partner="tweetdeck"><p lang="en" dir="ltr">Do you run an open source project with a vibrant community of contributors? I’d like to talk to you.</p>&mdash; Coraline Ada Ehmke (@CoralineAda) <a href="https://twitter.com/CoralineAda/status/723534302550593536">April 22, 2016</a></blockquote>
+
+
+[The US government and open source](http://www.linuxjournal.com/content/us-government-and-open-source-software)
+
+[STEMM Role Models](https://www.mozillascience.org/projects/KirstieJane-STEMMRoleModels) is a website that will help conference organizers connect with presenters from underrepresented groups.
+
+[How to become an advanced contributor to OpenStack](https://opensource.com/business/16/4/openstack-summit-interview-ildiko-vancsa)
+
+
+## The Web and Development
+
+[Don't trust your CMS](http://nymag.com/following/2016/04/dont-trust-your-cms.html)
+
+<blockquote class="twitter-tweet" data-partner="tweetdeck"><p lang="en" dir="ltr">Anyway next time someone says improving the CMS&#39;s UX is stupid and authors are just dumb, I&#39;ll print this article out and hit them with it.</p>&mdash; SaraWachterBoettcher (@sara_ann_marie) <a href="https://twitter.com/sara_ann_marie/status/723249259026501635">April 21, 2016</a></blockquote>
+
+
+Remember Glitch? [Some developers are trying to bring it back](http://elevengiants.com/).
+
+<blockquote class="twitter-tweet" data-partner="tweetdeck"><p lang="en" dir="ltr">code is art and you cannot tell me otherwise</p>&mdash; quantum computer (@Charlotteis) <a href="https://twitter.com/Charlotteis/status/723148411139198976">April 21, 2016</a></blockquote>
+
+[FreeCodeCamp abandoned their blog for Medium, and thinks you should too](https://medium.freecodecamp.com/we-just-abandoned-our-blog-for-medium-you-probably-should-too-33e742a1d49#.1btwyvm6u)
+
+
+## The Tech World and Culture
```

Comment:
Should we put the quotes first and then the linked article? This section reads confusing to me.